ftw/footer/footer_viewlet.py

```diff
@@ -52,15 +55,20 @@ def get_managers(self):
 
         def generate_classes(self, manager):
             width = self.calculate_width()
-            index = self.calculate_index(manager)
-            classes = 'column cell position-%s width-%s' % (index, width)
+            if IS_PLONE_5:
+                classes = 'col-lg-{}'.format(width)
+            else:
+                index = self.calculate_index(manager)
+                classes = 'column cell position-%s width-%s' % (index, width)
```

Comment:
Could you use format here ("new way"), or at least do it consistent within your own commit ;-)

Reply:
Fair point